setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-exclude = migrations
+exclude = migrations, snapshots
```

Comment:
Will do. Just tried to pass flake8 here since snapshots data sometimes generated too long lines that black failed to reformat them, especially when it comes to multiple lines string `""" """`